integration_tests/web/test_conversations.py

```diff
@@ -0,0 +1,129 @@
+import logging
```

Comment:
As this file has only tests for Slack Connect APIs, how about naming it to `test_conversations_connect.py`? If we will have integration tests for other conversations.* endpoints in the future, we can have them in a different file.